deepchecks/tabular/checks/data_integrity/data_duplicates.py

```diff
@@ -118,3 +122,30 @@ def run_logic(self, context: Context, dataset_kind):
             display = None
 
         return CheckResult(value=percent_duplicate, display=display)
+
+    @docstrings
+    @merge_args(SingleDatasetCheck.run)
+    def fix(self, *args, keep: Literal['first', 'last', False] = 'first', **kwargs) \
+            -> FixResult:
+        """Run fix.
+
+        Parameters
+        ----------
+        {additional_context_params:2*indent}
+        keep : Literal['first', 'last', False], default: 'first'
+            Whether to keep the first or last duplicate row.
+            If False, all duplicates will be removed.
+
+        Returns
+        -------
+        Dataset
+            Dataset with fixed duplicates.
+        """
+        context = self.get_context(*args, **kwargs)
```

Comment:
if i created a single check object then ran it multiple times the context will be overrided no?

Maybe a safer option is to make this method static